unstract/connectors/src/unstract/connectors/databases/mysql_handler.py

```diff
@@ -13,6 +14,26 @@
 
 
 class MysqlHandler:
+    @staticmethod
+    def sql_to_db_mapping(value: str) -> str:
+        """Function to generate information schema of the corresponding table.
+
+        Args:
+            table_name (str): db-connector table name
+
+        Returns:
+            dict[str, str]: a dictionary contains db column name and
+            db column types of corresponding table
+        """
+        python_type = type(value)
+        mapping = {
+            str: "LONGTEXT",
+            int: "BIGINT",
+            float: "FLOAT",
+            datetime.datetime: "TIMESTAMP",
+        }
```

Comment:
NIT: Consider moving this `mapping` to a `constants` file or refactoring it into an `enum`. This approach would make the code cleaner and easier to maintain.

Reply:
@muhammad-ali-e let's keep it for now. Because it's connector have different mapping